apps/designer/app/designer/features/style-panel/shared/css-value-input/css-value-input.tsx

```diff
@@ -309,13 +309,13 @@ export const CssValueInput = ({
   });
 
   const handleOnBlur: KeyboardEventHandler = (event) => {
+    inputProps.onBlur(event);
     // When select is open, onBlur is triggered,though we don't want a change event in this case.
-    if (isUnitsOpen || isOpen) {
+    if (isUnitsOpen || (isOpen && !getMenuProps().empty)) {
```

Comment:
MB anyone knows isOpen is always true when we are editing input. Independently does menu has items or not,
is it solvable in downshift?

Reply:
Looks like strange default behavior,  semantically its isFocused

Reply:
and yes the dropdown may be not empty even when not shown

Reply:
> When select is open, onBlur is triggered

Maybe that's where we broke something? In codesanbox above input does not loose focus when downshift opens

Reply:
Here 2 comboboxes. Comment is for 2nd. 
Will split on 2 ifs with different comments

Reply:
Splitted